test/DotNetEnv.Tests/ParserTests.cs

```diff
@@ -238,8 +249,22 @@ public void ParseUnquotedValue ()
             // TODO: is it possible to get the system to recognize when a complete unicode char is present and start the next one then, without a space?
 //            Assert.Equal("日本", Parsers.UnquotedValue.Parse(@"\xe6\x97\xa5\xe6\x9c\xac"));
 
-            Assert.Throws<ParseException>(() => Parsers.UnquotedValue.End().Parse("0\n1"));
-            Assert.Throws<ParseException>(() => Parsers.UnquotedValue.End().Parse("'"));
+            Assert.Equal("0", Parsers.UnquotedValue.Parse("0\n1").Value);   // value ends on linebreak
+            Assert.Equal("", Parsers.UnquotedValue.Parse("'").Value);  // unmatched singleQuote
```

Comment:
I am not sure that this is how this should work -- it seems like this should be an error to me. Do you feel differently / do you see other examples in here that you think makes this consistent with existing behaviors?

To be clear: I don't remember all the examples in here very well at this point, so it may very well be the case that this is entirely consistent and I am thinking about things all wrong. :)

Reply:
Due to rebasing I'm not completely sure if that's just a dupe, but I will look through the changes if there are more of these tests and reintroduce them appropriatly.

Reply:
Checked it, there are no other occurences.